examples/plug-and-play/demo_vanilla_PnP.py

```diff
@@ -134,8 +134,8 @@
 )  # reconstruction with PnP algorithm
 
 # compute PSNR
-print(f"Linear reconstruction PSNR: {dinv.utils.metric.cal_psnr(x, x_lin):.2f} dB")
-print(f"PnP reconstruction PSNR: {dinv.utils.metric.cal_psnr(x, x_model):.2f} dB")
+print(f"Linear reconstruction PSNR: {dinv.metric.PSNR()(x, x_lin).item():.2f} dB")
```

Comment:
shouldn't be dinv.loss.metric.PSNR?

Reply:
you can import directly with dinv.metric.PSNR, see above